src/GafferSceneUI/SceneGadget.cpp

```diff
@@ -237,14 +237,14 @@ const Gaffer::Context *SceneGadget::getContext() const
 	return m_controller->getContext();
 }
 
-void SceneGadget::setExpandedPaths( const IECore::PathMatcher &expandedPaths )
+void SceneGadget::setVisibleSet( const GafferScene::VisibleSet &visibleSet )
 {
-	m_controller->setExpandedPaths( expandedPaths );
+	m_controller->setVisibleSet( visibleSet );
```

Comment:
Likewise, this won't compile without the next commit.

Reply:
As above, squashed in with a67e1ed.